Client/Frontend/Browser/TabManager.swift

```diff
@@ -110,6 +110,26 @@ class TabManager: NSObject {
         return tabs.filter { $0.isPrivate }
     }
 
+    var recentlyAccessedNormalTabs: [Tab] {
+        assert(Thread.isMainThread)
```

Comment:
Curious about this assert. Have you come across a case where this block of code was accessed off main?

Reply:
No, but all the other "get tabs" functions in the TabManager had this, so I assumed there was a reason for it being there.